libbs/decompilers/ghidra/hooks.py

```diff
@@ -71,12 +72,21 @@ def domainObjectChanged(self, ev):
                     # Currently unused and unsupported
                     pass
                 elif changeType in symChgEvents:
-                    if obj == None and newValue != None:
+                    if obj is None and newValue is not None:
                         obj = newValue
                     if "VariableSymbolDB" in str(type(obj)):
-                        if oldValue and newValue:
-                            stackVar = StackVariable(None, newValue, None, None, None)
-                            self._interface.stack_variable_changed(stackVar)
+                        if oldValue and newValue and obj.parentNamespace is not None:
+                            self._interface.stack_variable_changed(
```

Comment:
@Flipout50 take note of a few things here:
1. When use the callback `x_changed(...)`, you must pass it a lifted form
2. Although most attributes for an Artifact should be `None`, there are a few that are 100% mandatory for indexing and storage of that artifact. In Functions that's their `addr`. In StackVar that's their `addr` and `stack_offset`. You must provide them when passing it off to the change function